coffee/controller/list-item-ctrl.coffee

```diff
@@ -1,7 +1,36 @@
 ListItemCtrl = ($scope, githubService) ->
 
-  $scope.getNotifications = () ->
-    githubService.notifications($scope.repo)
+  # Push notification to server
+  $scope.getNotifications = (repo) ->
+    nof = githubService.notifications(repo)
+    nof.then (notification) -> 
+      repo.notification = notification;
+    
+  # Get colors  
+  $scope.getColors = () ->
+    $scope.colors = $scope.globalState.user.getColors();
+
+  # Return color for every single language
+  $scope.languageColors = (languageName) ->
+      getColor =  $scope.colors[languageName]
+      # If the language inside array colors then return color of array
+      # If not keep using default color (Check class .color in _git_repo.scss)
+      if getColor 
+        return {
+          'background-color': getColor.color
+        } 
+
+  # Return text 'Updated on...'
+  $scope.updateOn = (timeText) ->
```

Comment:
instead of doing this,
we could use the angular built in date filter:
`ng-bind="repo.updated_at | date:'dd MMM yyyy'"`